include/koa/authentication/passport-strategies/strategy_services.js

```diff
@@ -0,0 +1,93 @@
+function _getDao(loginContext, pb) {
+    if (loginContext.site) {
+        return new pb.SiteQueryService({
+            site: loginContext.site,
+            onlyThisSite: false
+        });
+    } else {
+        return new pb.DAO();
+    }
+};
+
+function _addUserToSession(req, user, pb) {
+    delete user.password;
+
+    //build out session object
+    user.permissions = pb.PluginService.getPermissionsForRole(user.admin);
+    req.session.authentication.user = user || {};
+    req.session.authentication.user_id = user[pb.DAO.getIdField()] ? user[pb.DAO.getIdField()].toString() : null;
```

Comment:
Does this function give a consistent result that actually nets us a useful value?  I find that shocking.

Reply:
or is that the thing that adds _id to the object? 🤷‍♂️

Reply:
Yes, it only adds the _id to the object :P Is the old function from the old core.